components/suggest/src/suggestion.rs

```diff
@@ -16,6 +16,12 @@ const TIMESTAMP_TEMPLATE: &str = "%YYYYMMDDHH%";
 /// 2 bytes shorter than [`TIMESTAMP_TEMPLATE`].
 const TIMESTAMP_LENGTH: usize = 10;
 
+#[derive(Clone, Debug, PartialEq)]
+pub struct SuggestionIcon {
+    pub data: Vec<u8>,
+    pub mimetype: String,
```

Comment:
Hah, I was just about to suggest `mime_type`, and then saw you renamed it for consistency with the schema.

I think `mime_type` looks a little cleaner (it becomes `mimeType` in JS, Swift, and Kotlin; `mimetype` there almost looks like a typo). I wouldn't worry too much about keeping it consistent with the schema—we already have other fields that use different names! ☺️

Reply:
Okay! I will change to `mime_type`.